vm/systemSmartContracts/auction.go

```diff
@@ -308,7 +310,25 @@ func (s *stakingAuctionSC) getVerifiedBLSKeysFromArgs(txPubKey []byte, args [][]
 	return blsKeys
 }
 
+func (s *stakingAuctionSC) isFunctionEnabled(nonce uint64) bool {
+	currentNonce := s.eei.BlockChainHook().CurrentNonce()
+	if currentNonce == 0 {
+		return true
+	}
+
+	if currentNonce < nonce {
```

Comment:
can simply use `return currentNonce >= nonce`

Reply:
implemented completely the function.